crates/bevy_gilrs/src/converter.rs

```diff
@@ -1,7 +1,7 @@
 use bevy_input::gamepad::{Gamepad, GamepadAxisType, GamepadButtonType};
 
 pub fn convert_gamepad_id(gamepad_id: gilrs::GamepadId) -> Gamepad {
-    Gamepad(gamepad_id.into())
+    Gamepad::new(gamepad_id.into())
```

Comment:
Could this be an implementation of From::\<GamepadId\> instead? Seems more idiomatic

Reply:
I don't think that we could do it that easily, because we would need a `gilrs` dependency inside of `bevy_input` to write that `impl`. Having the `gilrs` stuff separate seems more reasonable to me and therefore the function should be fine. If `GamepadId` was a `bevy` type I would definitely agree with you on this one!